packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -113,7 +113,7 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		useBlockRefProvider( clientId ),
 		useFocusHandler( clientId ),
 		useEventHandlers( { clientId, isSelected } ),
-		useZoomOutModeExit( { editorMode } ),
+		useZoomOutModeExit( { clientId, editorMode } ),
```

Comment:
Looks like editorMode is dropped in #65326, so this probably needs a rebase.

Reply:
👍